src/library/scala/collection/immutable/Range.scala

```diff
@@ -354,6 +354,35 @@ sealed abstract class Range(
       else head
     } else super.max(ord)
 
+  override def tails: Iterator[Range] =
```

Comment:
Smart!

Reply:
Shouldn't `tails` return the empty iterator for an empty range?

Reply:
I don't change behavior.

```
Welcome to Scala 2.12.6 (Java HotSpot(TM) 64-Bit Server VM, Java 1.8.0_181).
Type in expressions for evaluation. Or try :help.

scala> Nil.tails.toList
res0: List[List[Nothing]] = List(List())

scala> (1 until 1).tails.toList
res1: List[scala.collection.immutable.IndexedSeq[Int]] = List(Vector())
```

```
Welcome to Scala 2.13.0-M5 (Java HotSpot(TM) 64-Bit Server VM, Java 1.8.0_181).
Type in expressions for evaluation. Or try :help.

scala> Nil.tails.toList
res0: List[List[Nothing]] = List(List())

scala> (1 until 1).tails.toList
res1: List[IndexedSeq[Int]] = List(Vector())
```

Reply:
@xuwei-k I stand corrected, Yoshida-san! :)